### PR TITLE
Support eager loading translations 2.0

### PR DESCRIPTION
**Pull request by alepeino**
(Just another try at merging #71)

### CODE REVIEW

**Comment by vinkla** [comment]
Great! Could you also add this to the documentation?

File: src/Translatable.php [hunk 1]
```diff
@@ -33,6 +34,23 @@
     protected $cache = [];
 
     /**
+     * Query scope for eager-loading the translations for current (or a given) locale.
+     *
+     * @param \Illuminate\Database\Eloquent\Builder $builder
+     * @param string $locale
```
Comment (vinkla) [inline src/Translatable.php]: This should be `string|null`.

File: src/Translatable.php [hunk 2]
```diff
@@ -90,7 +108,7 @@ protected function getTranslation(string $locale)
             return $this->cache[$locale];
         }
 
-        $translation = $this->translations()
+        $translation = $this->translations
```
Comment (vinkla) [inline src/Translatable.php]: Did you forget `relationLoaded`?

Comment (alepeino) [inline src/Translatable.php]: Actually, `Model` base class handles that via `__get` method!

Comment (vinkla) [inline src/Translatable.php]: Oh, didn't know about that.

**Comment by vinkla** [comment]
Thanks for the pull request!

**Comment by vinkla** [comment]
I've released the changes in 4.2.0. Thanks for helping out!

**Comment by alepeino** [comment]
Great, I look forward to keep contributing!

**Pull request merged**